### DISCUSSION TITLE
DataTable : Setting column widths with resizable

### DISCUSSION THREAD

**Discussion by gabrielsaliba1987** [primefaces/community#1958] [PrimeVue]
Hi, we are using the DataTable component  and our requirement is to have resizable columns but also predefine specific set of column width for certain columns. 

For example : 

- Title : 20px (non resizable)
- Name : auto (resizable)
- Surname : auto (resizable)
- Address : 500px (resizable)

How can I be able to achieve to this because I cannot figure out how can I pass these parameters in the column template. 

I have search quite a bit even posts dating back to 2016 I cannot figure this out.

**Comment by gabrielsaliba1987** [comment]
Any help about this?! No one?

**Comment by pickbreaker** [comment]
I was searching for the same problem, when I stumbled upon your question. I don't believe this is currently possible but I need this functionality too. Unfortunately there does not seem to be a way in which we can currently access the column widths.
